tests/Unit/PointwiseFunctions/Hydro/Test_TransportVelocity.cpp

```diff
@@ -0,0 +1,38 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <cmath>
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/TransportVelocity.hpp"
+#include "tests/Unit/Pypp/CheckWithRandomValues.hpp"
+#include "tests/Unit/Pypp/SetupLocalPythonEnvironment.hpp"
+
+namespace hydro {
+namespace {
+template <size_t Dim, typename Frame, typename DataType>
+void test_transport_velocity(const DataType& used_for_size) {
+  pypp::check_with_random_values<1>(&transport_velocity<DataType, Dim, Frame>,
```

Comment:
Could the transport velocity function be checked against an analytic solution too?